### DISCUSSION TITLE
When motion detected. Pic and Video has Text (Trigger, Postcap)

### DISCUSSION THREAD

**Discussion by Bkk20** [Motion-Project/motion#1719] [Q&A]
I cannot find any solution for that but I see text information in my pic and video like "motion", "trigger" "postcap" with Timing. How can I remove it. What kind of function is it?

**Comment by Bkk20** [comment]
Does now one has any idea. The text are shown in the left upper area. First I see text "motion" then"trigger" then" "postcap" and "precap". Below this is the date and time.
I want to remove it. How?

**Comment by Mr-Dave** [comment]
I would guess that you have `setup_mode on` as a configuration option.  Change it to `setup_mode off`

**Comment by Bkk20** [comment]
Hi, thanks for your idea but setup_mode is off. Some other settings has to integrate the text in my video and picture.

**Comment by Bkk20** [comment]
![6-01-2023-10-26_210314-15](https://github.com/Motion-Project/motion/assets/90546929/97c16fbe-7ab9-4091-bb5e-fee6fed75fcd)

**Reply by Bkk20** [reply]
This is the information. The text changes to "trigger" then" "postcap" and "precap" in the video. Depends what motion is doing.

**Comment by Mr-Dave** [comment]
Change log_level

**Reply by Bkk20** [reply]
Omg, yes! That's the solution. Thanks 😊
